commitlint/helpers.ts

```diff
@@ -42,12 +42,13 @@ export abstract class Helpers {
         return text as string;
     }
 
-    public static assertWhen(when: string) {
+    public static assertWhen(when: When) {
```

Comment:
@realmarv this change is wrong, when should be string, but the return type should be When (I told you to look at how we already do this in assertNotNull()

Reply:
fixed it